dw-menu-list-item.js

```diff
@@ -163,7 +164,7 @@ class DwMenuListItem extends LitElement {
           return html`<dw-menu-list-item
             .action=${action}
             .level=${this.level + 1}
-            ?hasLeadingIcon=${this.action.subAction.some((e) => e.icon)}
+            ?hasLeadingIcon=${this.action.subAction.some((e) => e.icon || e.hasLeadingIconSpace)}
```

Comment:
Do we need this feature for child items only? If yes, add it to the documentation.